README.md

```diff
@@ -68,6 +69,19 @@ Commands:
 
 ### Details
 
+#### archive
+
+```
+$ apiary help archive
+Usage:
+  apiary archive
+
+Options:
+  [--output=FILE]  # Write API Description Document into specified file
```

Comment:
If I read the code correctly this doesn't seem to be supported.

Reply:
I will try remove this option.

Reply:
Do you think that make sense make some output path option there?

Reply:
I removed this option now.